spec/spec_helper.rb

```diff
@@ -21,6 +21,27 @@
 
 require 'webmock/rspec'
 
+Unleash.configure do |config|
+  config.url      = 'http://default-test-url/'
+  config.app_name = 'default-my-test-app'
+end
+
+default_features = {
+  "version": 1,
+  "features": [
+    {
+      "name": "Feature.A",
+      "description": "Enabled toggle",
+      "enabled": true,
+      "strategies": [{ "name": "default" }]
+    }
+  ]
+}
+
+WebMock.stub_request(:get, "http://default-test-url/client/features").to_return(status: 200, body: default_features.to_json, headers: {})
+
+Unleash.logger = Unleash.configuration.logger
```

Comment:
I would prefer to keep these in the the scope of `toggle_fetcher_spec.rb` for now, as it is only used there, and can be confusing for others later when looking at the test code of what is relevant to which _spec tester.

Eventually it would make sense to have some of most common settings in a common configuration file, but I see that as a separate PR.

Reply:
Ok, makes sense. Moved the setup to the spec itself.